webextensions/api/devtools.json

```diff
@@ -611,6 +914,65 @@
               }
             }
           },
+          "SourcesPanel": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": true
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            },
+            "createSidebarPane": {
+              "__compat": {
+                "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools/panels/SourcesPanel/createSidebarPane",
+                "support": {
+                  "chrome": {
+                    "version_added": true
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "onSelectionChanged": {
+              "__compat": {
+                "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools/panels/SourcesPanel/onSelectionChanged",
+                "support": {
+                  "chrome": {
+                    "version_added": true
```

Comment:
This appears to be from before Chrome 21 as the reference to this file existed before the oldest historical entry on [chrome/common/extensions/api/devtools/panels.json](https://source.chromium.org/chromium/chromium/src/+/main:chrome/common/extensions/api/devtools/panels.json;bpv=1).